js/cta.js

```diff
@@ -74,7 +74,7 @@ function renderArticle(article, section, number) {
     article.tags = processTags(article.tags);
     var sectionSelector = "#article" + number;
 
-    var articleHref = "./" + article.url.slice(article.url.indexOf("org") + 4);
+    var articleHref = "./" + article.url.slice(article.url.indexOf("in") + 3);
```

Comment:
```suggestion
    var articleHref = "./" + article.url.slice(article.url.indexOf(".in") + 4);
```
Looks fine as a temporary fix.
Lets slice from ".in" to prevent other "in" which may occur as a substring. @mahim23  WDYT?